pkg/apis/dashboard/extension.go

```diff
@@ -185,26 +183,43 @@ func getServiceRevisionStatusStats(
 		return nil, err
 	}
 
-	err = query.
-		Where(
-			servicerevision.CreateTimeGTE(startTime),
-			servicerevision.CreateTimeLTE(endTime)).
-		Modify(func(q *sql.Selector) {
-			// Count.
-			q.
-				Select(
-					sql.As(sql.Count(servicerevision.FieldStatus), "count"),
-					sql.As(groupBy, servicerevision.FieldCreateTime),
-					servicerevision.FieldStatus).
-				GroupBy(
-					groupBy,
-					servicerevision.FieldStatus)
-		}).
-		Scan(ctx, &counts)
+	// Ent doesn't support group by json field yet.
```

Comment:
Can you try the following?
```
q.
Select(
	sql.As(sql.Count(servicerevision.FieldStatus), "count"),
	sql.As(groupBy, servicerevision.FieldCreateTime),
	sql.As("status ->> 'summaryStatus'", "summaryStatus")).
GroupBy(
	servicerevision.FieldCreateTime,
	"summaryStatus")
```
Let's avoid using raw sql if possible.

Reply:
Not working. error message column "status ->> 'summaryStatus'" does not exist